masterlist.yaml

```
@@ -7915,6 +7915,48 @@ plugins:
   - name: 'Improved dungeons.esp'
     msg: [ *obsolete ]
 
+  - name: '(Mage Equipment|Francesco''s Mage Equipment)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/18875/'
+        name: 'Mage Equipment 21'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Mage Equipment 21 ESP' ]
+        condition: 'many("(Mage Equipment|Francesco''s Mage Equipment)\.esp")'
```

The regex can be simplified a bit.

```suggestion
  - name: '(Francesco''s )?Mage Equipment\.esp'
    url:
      - link: 'https://www.nexusmods.com/oblivion/mods/18875/'
        name: 'Mage Equipment 21'
    msg:
      - <<: *useOnlyOneX
        subs: [ 'Mage Equipment 21 ESP' ]
        condition: 'many("(Francesco''s )?Mage Equipment\.esp")'
```